lib/modules/auth/signin.dart

```diff
@@ -0,0 +1,209 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:podcasty/modules/auth/signup.dart';
+import 'package:podcasty/modules/auth/widgets/form_header.dart';
+import 'package:podcasty/modules/auth/widgets/header_widget.dart';
+import 'package:podcasty/values/app_theme.dart';
+import 'package:podcasty/values/string_constants.dart';
+import 'package:podcasty/widgets/custom_button.dart';
+import 'package:podcasty/widgets/input_field.dart';
+
+import '../../values/app_colors.dart';
+
+class SignInScreen extends StatefulWidget {
+  const SignInScreen({Key? key}) : super(key: key);
+
+  @override
+  SigninForm createState() => SigninForm();
+}
+
+class SigninForm extends State<SignInScreen> {
+  final signinFormKey = GlobalKey<FormState>();
+  bool isObscure = true;
+  bool rememberMe = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: SafeArea(
+      bottom: false,
+      child: SingleChildScrollView(
+        physics: const BouncingScrollPhysics(),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: [
+            const HeaderWidget(
+              isLeftOriented: false,
+            ),
+            Container(
+              height: MediaQuery.of(context).size.height * 0.8 + 16,
+              decoration: const BoxDecoration(
+                color: AppColors.blueGrey,
+                borderRadius: BorderRadius.only(
+                  topLeft: Radius.circular(130.0),
+                ),
+              ),
+              child: Padding(
+                padding: EdgeInsets.symmetric(
+                    horizontal: MediaQuery.of(context).size.width * 0.04),
```

Comment:
Please avoid repeated calls for inherited widget.

Reply:
Resolved